components/eventing-controller/controllers/subscription/beb/reconciler_test.go

```diff
@@ -271,7 +271,7 @@ var _ = Describe("Subscription Reconciliation Tests", func() {
 				eventingv1alpha1.ConditionAPIRuleStatus,
 				eventingv1alpha1.ConditionReasonAPIRuleStatusNotReady,
 				v1.ConditionFalse,
-				"",
+				"sink URL scheme should be 'http' or 'https'",
```

Comment:
Are we sure that sink url can be `https`?

Reply:
This is the same error message from [here](https://github.com/kyma-project/kyma/blob/cc5af9ca8fae62d9166896554bf83615272c0fe5/components/eventing-controller/pkg/handlers/sink/validator.go#L21). But I will see if I can use that variable directly instead :)